cmake/modules/TmpCPack.cmake

```diff
@@ -0,0 +1,24 @@
+set(CPACK_PACKAGE_DIRECTORY "${CMAKE_BINARY_DIR}/_packages")
```

Comment:
We don't need it here, the default value is sane and can be overridden with `cpack -B...`